### ISSUE TITLE
人脸检测gpu部署mkldnn=true，报错

### ISSUE THREAD

**Issue by stefan252423** [PaddlePaddle/PaddleX#1261]
## 描述问题
   人脸检测，window环境cpu推理，当打开mkldnn加速后，报错，关闭后正常运行，不知道是什么原因造成的？
## 复现

1. c++部署方式
     windows环境，gpu部署

2. 您使用的**模型**和**数据集**是？
    使用的是BlazeFace-FPN-SSH人脸检测模型：地址 ：
                                 https://github.com/deepinsight/insightface/tree/master/detection/blazeface_paddle
    一张随便的照片：                
![test](https://user-images.githubusercontent.com/68883664/142754075-c73d81a7-0a0f-42ad-8819-10c41389b2ed.jpg)

3. 请提供您出现的报错信息及相关log
![image](https://user-images.githubusercontent.com/68883664/142753723-0219e677-1cff-4f1a-8ab3-512df5c9003c.png)
 e[37m---    Fused 0 ReshapeTransposeMatmulMkldnn patterns with reshape's xshapee[0m
e[37m---    Fused 0 ReshapeTransposeMatmulMkldnn patterns with reshape's xshape with transpose's xshapee[0m
e[32m--- Running IR pass [matmul_transpose_reshape_fuse_pass]e[0m
e[37m---    Fused 0 MatmulTransposeReshape patternse[0m
e[32m--- Running IR pass [batch_norm_act_fuse_pass]e[0m
e[37m---    fused 0 batch norm with relu activatione[0m
e[1me[35m--- Running analysis [ir_params_sync_among_devices_pass]e[0m
e[1me[35m--- Running analysis [adjust_cudnn_workspace_size_pass]e[0m
e[1me[35m--- Running analysis [inference_op_replace_pass]e[0m
e[1me[35m--- Running analysis [ir_graph_to_program_pass]e[0m
I1121 15:29:06.352128 17188 analysis_predictor.cc:636] ======= optimize end =======
I1121 15:29:06.352128 17188 naive_executor.cc:98] ---  skip [feed], feed -> scale_factor
I1121 15:29:06.353127 17188 naive_executor.cc:98] ---  skip [feed], feed -> image
I1121 15:29:06.353127 17188 naive_executor.cc:98] ---  skip [feed], feed -> im_shape
I1121 15:29:06.362099 17188 naive_executor.cc:98] ---  skip [save_infer_model/scale_0.tmp_1], fetch -> fetch
I1121 15:29:06.362099 17188 naive_executor.cc:98] ---  skip [save_infer_model/scale_1.tmp_1], fetch -> fetch
I1121 15:29:06.386075 17188 device_context.cc:598] oneDNN v2.2.1
W1121 15:29:06.528693 17188 operator.cc:242] elementwise_add raises an exception struct dnnl::error, could not create a descriptor for a binary operation primitive

## 环境

4. PaddleX分支：develop分支、推理引擎版本号：cpu_avx_mkl v2.1.1

5. 请提供您使用的操作系统信息：Windows x64 core i7 10th

**Comment by FlyingQianMM** [comment]
试下推理引擎2.2吧，看下是否已经解决了

**Comment by stefan-567** [comment]
我在如下网站（https://paddleinference.paddlepaddle.org.cn/user_guides/download_lib.html#windows）只看到了windows 2.1.1版本的依赖库，麻烦发我一个2.2版本的依赖库下载网址，多谢。

**Comment by stefan-567** [comment]
> 试下推理引擎2.2吧，看下是否已经解决了

我在如下网站（https://paddleinference.paddlepaddle.org.cn/user_guides/download_lib.html#windows）只看到了windows 2.1.1版本的依赖库，麻烦发我一个2.2版本的依赖库下载网址，多谢。

**Comment by stefan252423** [comment]
> 试下推理引擎2.2吧，看下是否已经解决了

好的，我去试试，多谢
